bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -178,17 +169,7 @@ spec:
             - -c
             - |
               set -xe
-              # Determine the ovn rundir.
-              if [[ -f /usr/bin/ovn-appctl ]] ; then
-                  # ovn-appctl is present. Use new ovn run dir path.
-                  DB_SOCK_PATH=/var/run/ovn/ovnnb_db.ctl
-                  APPCTL_PATH=/usr/bin/ovn-appctl
-              else
-                  # ovn-appctl is not present. Use openvswitch run dir path.
-                  DB_SOCK_PATH=/var/run/openvswitch/ovnnb_db.ctl
-                  APPCTL_PATH=/usr/bin/ovs-appctl
-              fi
-              exec ${APPCTL_PATH} -t ${DB_SOCK_PATH} cluster/status OVN_Northbound  2>/dev/null | grep ${K8S_NODE_IP} | grep -v Address -q
+              exec /usr/bin/ovn-appctl -t /var/run/ovn/ovnnb_db.ctl ovsdb-server/sync-status
```

Comment:
you're reverting back to the older healthcheck command here

Reply:
fixed